### PR TITLE
Add alert flow integration test

### PR DESCRIPTION
**Pull request by iskhakov**
**What this PR does**:
Adds pytest integration test which
Tests celery tasks related to the alert lifecycle
This test is celery specific, see https://docs.celeryq.dev/en/stable/userguide/testing.html
It uses in memory broker and runs celery worker in separate thread

**Which issue(s) this PR fixes**:

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated

### CODE REVIEW

File: engine/apps/alerts/tests/integration_test_alert_flow.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+from apps.alerts.models import AlertGroup, AlertReceiveChannel
+from apps.integrations.tasks import create_alert
+
+pytest_plugins = ["celery.contrib.pytest"]
+
+# Uncomment this block if you want to test specific queues
+# @pytest.fixture(scope='session')
+# def celery_worker_parameters():
+#     return {
+#         'queues':  ('celery', 'default', 'retry', 'critical'),
```
Comment (joeyorlando) [inline engine/apps/alerts/tests/integration_test_alert_flow.py:15]: is this commented out block still required?

File: engine/apps/alerts/tests/integration_test_alert_flow.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+#     }
+
+
+@pytest.fixture(scope="session")
```
Comment (joeyorlando) [inline engine/apps/alerts/tests/integration_test_alert_flow.py:19]: `scope="session"` means that this fixture is auto-used once for **all** tests, correct? could this maybe be scoped to this file (`scope="module"`)? or if it makes sense to keep it to `scope="session"`, maybe move it to `engine/conftest.py`?

**Review by joeyorlando — APPROVED** [review]
Nice 👍 

out of curiosity, since the file is named `integration_test_*`, will this test file be picked up when running `pytest` or does it need to be run separately?

**Comment by iskhakov** [comment]
not relevant anymore

**Pull request closed without merging**